config/database.php

```diff
@@ -157,6 +157,12 @@
             'database' => env('REDIS_CACHE_DB', 1),
         ],
 
+        'metrics' => [
+            'host' => env('REDIS_HOST', '127.0.0.1'),
+            'password' => env('REDIS_PASSWORD', null),
+            'port' => env('REDIS_PORT', 6379),
+            'database' => env('REDIS_METRICS_DB', 2),
```

Comment:
This is not supported by the Prometheus library that just uses a hardcoded `PROMETHEUS_` prefix on keys. I shall remove the setting again.